test/helpers/lightning.ts

```diff
@@ -18,9 +18,9 @@ import {
   updateRoutingFees,
 } from "lightning"
 
-import { bitcoindClient, RANDOM_ADDRESS } from "./bitcoinCore"
+import { bitcoindClient, bitcoindOutside, RANDOM_ADDRESS } from "./bitcoinCore"
 
-export * from "lightning"
+export * from "lightning" // TODO?
```

Comment:
why adding TODO?

Reply:
Right that was something I wanted to ask, but then after listening to @dolcalmi I am more clear about its purpose. Will remove the comment.